android/src/toga_android/app.py

```diff
@@ -239,3 +240,6 @@ def hide_cursor(self):
 
     def show_cursor(self):
         pass
+
+    def get_screens(self):
+        return [ScreenImpl(self.interface.main_window._impl)]
```

Comment:
This is using an impl class as the native. I'm not sure that's the right abstraction; if nothing else, there's [DisplayManager](https://developer.android.com/reference/android/hardware/display/DisplayManager) which can return the list of attached displays.

Reply:
Modified it but not sure if it will work.